authentication/src/main/kotlin/com/uber/sdk2/auth/api/AuthProviding.kt

```diff
@@ -18,12 +18,15 @@ package com.uber.sdk2.auth.api
 import com.uber.sdk2.auth.api.response.AuthResult
```

Comment:
Out of scope of the PR, but -1 to that `api` in package name. It's just "noise". This is how I think about this: If I'm *importing* something from an external library, it's obviously a public API.

Reply:
fair point. any recommendations? how about core?

Reply:
I'd probably just drop `api`, so `com.uber.sdk2.auth.response.AuthResult`